apps/web/src/components/Menu/index.tsx

```diff
@@ -28,6 +30,9 @@ const Menu = (props) => {
   const cakePrice = useCakePrice()
   const { currentLanguage, setLanguage, t } = useTranslation()
   const { pathname } = useRouter()
+  const { selectUserfeatureABResult } = useABTestingManager()
+  const isNotificationFeatureEnabled = selectUserfeatureABResult(FEATURE_FLAGS.WebNotifications)
```

Comment:
still can simplify the usage. Ideally

const enabled = useFeatureEnabled(featureSelector)